h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1022,16 +1027,16 @@ public Chunk compress() {
     Chunk res = compress2();
     byte type = type();
     assert _vec == null ||  // Various testing scenarios do not set a Vec
-      type == _vec._type || // Equal types
-      // Allow all-bad Chunks in any type of Vec
-      type == Vec.T_BAD ||
-      // Specifically allow the NewChunk to be a numeric type (better be all
-      // ints) and the selected Vec type an categorical - whose String mapping
-      // may not be set yet.
-      (type==Vec.T_NUM && _vec._type==Vec.T_CAT) ||
-      // Another one: numeric Chunk and Time Vec (which will turn into all longs/zeros/nans Chunks)
-      (type==Vec.T_NUM && _vec._type == Vec.T_TIME && !res.hasFloat())
-      : "NewChunk has type "+Vec.TYPE_STR[type]+", but the Vec is of type "+_vec.get_type_str();
+        type == _vec._type || // Equal types
+        // Allow all-bad Chunks in any type of Vec
+        type == Vec.T_BAD ||
+        // Specifically allow the NewChunk to be a numeric type (better be all
+        // ints) and the selected Vec type an categorical - whose String mapping
+        // may not be set yet.
+        (type==Vec.T_NUM && _vec._type==Vec.T_CAT) ||
+        // Another one: numeric Chunk and Time Vec (which will turn into all longs/zeros/nans Chunks)
+        (type==Vec.T_NUM && _vec._type == Vec.T_TIME && !res.hasFloat())
+        : "NewChunk has type "+Vec.TYPE_STR[type]+", but the Vec is of type "+_vec.get_type_str();
```

Comment:
can you turn that off and revert all files that have only cosmetic changes? we'd prefer to keep the history and have less overhead for the reviewers.